main.go

```diff
@@ -188,12 +189,15 @@ func prune(cli *client.Client, deathNote *sync.Map) {
 			log.Println(err)
 		} else {
 			for _, container := range containers {
-				cli.ContainerRemove(context.Background(), container.ID, types.ContainerRemoveOptions{RemoveVolumes: true, Force: true})
+				err := cli.ContainerRemove(context.Background(), container.ID, types.ContainerRemoveOptions{RemoveVolumes: true, Force: true})
+				if err != nil {
+					log.Println(err)
```

Comment:
Do we want to `continue` here to avoid incrementing the number of deleted containers?

Reply:
If I am nitpicking, I would probably add this in a follow-up bugfix PR, to have a slightly better Git history 🙂

Reply:
Mmmm I think it is actually part of the change: handle the error, which was not handled in the past

Reply:
Which number of deleted containers we would have before? Probably no output, since it would fail? Or we would have the wrong number?

Reply:
The error was swallowed, the execution continued adding a container to the map.

Being strict, it could be considered a fix, yeah

Reply:
But we can also be pragmatic and fix it here. I think having the discussion was more important than the detail where we fix it (as long as we fix it).

Reply:
I'm unsure what the expected functionality should be at this time, happy to discuss it further in another PR as there are a handful of unhandled errors.